From: Sophie Brun <sophie@offensive-security.com>
Date: Wed, 8 Jul 2020 10:56:49 +0200
Subject: Remove entry_point to install helper-script

Last-Update: 2020-07-08

Description: don't install entry_point but use helper-script to be in
the directory of the project and allow program to find the wordlists and
api-key.
This patch should probably be removed for the next upstream release.
---
 setup.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/setup.py b/setup.py
index 71c62f3..fa185f4 100755
--- a/setup.py
+++ b/setup.py
@@ -14,12 +14,12 @@
     long_description_content_type="text/markdown",
     url="https://github.com/laramies/theHarvester",
     packages=setuptools.find_packages(exclude=['tests']),
-    entry_points={
-        'console_scripts': [
-            'theHarvester = theHarvester.__main__:entry_point'
-        ]
-    },
-
+#    entry_points={
+#        'console_scripts': [
+#            'theHarvester = theHarvester.__main__:entry_point'
+#        ]
+#    },
+#
     classifiers=[
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.7",
